### PR TITLE
Add Global Mobilization compatibility

### PR DESCRIPTION
**Pull request by TheMagnetar**
**When merged this pull request will:**
- Title

### CODE REVIEW

**Comment by jonpas** [comment]
Need to modify license as discussed on Slack. Also need to adapt tools a bit (from ACE3) so we can publish optionals in a nice launchable way.

Will do later.

File: optionals/sys_gm/config.cpp [hunk 1]
```diff
@@ -0,0 +1,23 @@
+#include "script_component.hpp"
+
+class CfgPatches {
+    class ADDON {
+        name = COMPONENT_NAME;
+        units[] = {};
+        weapons[] = {};
+        requiredVersion = REQUIRED_VERSION;
+        requiredAddons[] = {"acre_main", "acre_sys_core"};
```
Comment (jonpas) [inline optionals/sys_gm/config.cpp]:
```suggestion
        requiredAddons[] = {"acre_main"};
```

File: optionals/sys_gm/config.cpp [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+        author = ECSTRING(main,Author);
+        authors[] = {"GlobalMobilization"};
```
Comment (jonpas) [inline optionals/sys_gm/config.cpp]:
```suggestion
        authors[] = {"Vertexmacht"};
```

File: optionals/sys_gm/config.cpp [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+        VERSION_CONFIG;
+    };
+};
+
+class CfgAcreWorlds {
+    class gm_weferlingen_summer {
+        wrp = "\idi\acre\addons\sys_gm\gm_weferlingen.fakewrp";
+    };
+
+    class gm_weferlingen_winter : gm_weferlingen_summer {};
```
Comment (jonpas) [inline optionals/sys_gm/config.cpp]:
```suggestion
    class gm_weferlingen_winter: gm_weferlingen_summer {};
```

Comment (TheMagnetar) [reply optionals/sys_gm/config.cpp]: Too much c++...

File: optionals/sys_gm/LICENSE [hunk 4]
```diff
@@ -2,7 +2,8 @@ Some items in this component (this folder and all subfolders)
 are licensed differently from the project's license or have
 additional terms of use!
 
-Following items may not be distributed outside of ACRE2 without
+Following files or folders with all its subfolders
```
Comment (TheMagnetar) [inline optionals/sys_gm/LICENSE]: The following files and/or folders

**Review by jonpas — DISMISSED** [review]

**Review by jonpas — APPROVED** [review]

**Pull request merged**